README.md

```diff
@@ -24,31 +24,68 @@ Create a new temporary project:
 * With a dependency that have a fixed version:
     `$ cargo-temp anyhow==1.0.13`
 
+### Repositories
+
+You can add repositories to you Cargo.toml.
+
+Examples:
+
+* HTTP
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git`
+
+* SSH
+    `$ cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git`
+
+This will add the repository on the default branch by default. You can choose
+another branch or a revision:
+
+* Branch
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=main`
```

Comment:
```suggestion
    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=master`
```
`anyhow` uses `master` as default branch. If you want to avoid `master` while still being consistent in your doc you could use another repository as example

Reply:
`master` is a good example too. i just didn't check

Reply:
It's not about good or better. It's about: is the example work? Did you actually run the command you are giving as example? Does it work?

In this case anyhow is no branch main so the example would fail. You have to assume a new user of cargo-temp will copy-paste your examples and they must work otherwise they will move on.